test/server/config_validation/xds_corpus/example1

```diff
@@ -0,0 +1,31 @@
+actions {
+  add_listener {
+    version : "1"
+    name : "listener_0"
+    route_config : "route_config_0"
+  }
+}
+actions {
+  add_route {
+    name : "route_config_0"
+    cluster : "cluster_0"
+  }
+}
+actions {
+  remove_route {
```

Comment:
I'm genuinely curious how we can remove a route config today, I was having this conversation with @stevenzzzz and with SoTW updates, this doesn't seem possible? :)